libraries/ui-shared/RatingStars/index.jsx

```diff
@@ -154,10 +154,10 @@ class RatingStars extends Component {
         aria-label={this.getTextualFinal(ratedStars)}
         data-test-id={`ratedStars: ${ratedStars}`}
       >
-        <div className={styles.emptyStars}>
+        <div className={`${styles.emptyStars} rating-stars-empty`}>
```

Comment:
These classes also need to be added to the ratings stars from the product-reviews extension

Reply:
good point